debian/nntrainer-dev.install

```diff
@@ -12,7 +12,6 @@
 /usr/include/nntrainer/tensor_v2.h
 /usr/include/nntrainer/tensor_base.h
 /usr/include/nntrainer/float_tensor.h
-/usr/include/nntrainer/half_tensor.h
```

Comment:
later pr, I think we also enable the fp16 support for debian build.

Reply:
for sure. I'll keep track of half_tensor.h when it comes